### ISSUE TITLE
Release 0.1.7?

### ISSUE THREAD

**Issue by joprice** [InTheNow/sbt-catalysts#23]
I can't resolve 0.1.7. I'm trying to upgrade from 0.1.6, where the groupId was "com.github.inthenow".

**Comment by kailuowang** [comment]
I expect 0.1.7 to be here [typelevel bintray](https://bintray.com/typelevel/sbt-plugins) but it seems empty
And if it's going to be published there. it might be helpful to add the extra resolver to the README?
```scala
resolvers += Resolver.url(
  "typelevel-sbt-plugins",
   url("http://dl.bintray.com/content/typelevel/sbt-plugins"))(
      Resolver.ivyStylePatterns)
```

**Comment by larsrh** [comment]
Resolved by #25. There's an `0.1.8` now on Bintray.
